trait bound `Postgres: sqlx_core::database::Database` is not satisfied

When running [this repo](https://github.com/PXNX/tartaros-telegram), I get the following error:

```
error[E0277]: the trait bound `Postgres: sqlx_core::database::Database` is not satisfied
  --> src/main.rs:51:13
   |
51 |     mut tx: Tx<Postgres>,
   |             ^^^^^^^^^^^^ the trait `sqlx_core::database::Database` is not implemented for `Postgres`
   |
   = help: the trait `sqlx_core::database::Database` is implemented for `sqlx_core::postgres::database::Postgres`
note: required by a bound in `Tx`
  --> /home/nyx/.cargo/registry/src/github.com-1ecc6299db9ec823/axum-sqlx-tx-0.3.0/src/tx.rs:75:19
   |
75 | pub struct Tx<DB: sqlx::Database, E = Error>(Lease<sqlx::Transaction<'static, DB>>, PhantomData<E>);
   |                   ^^^^^^^^^^^^^^ required by this bound in `Tx`
```
   
   Is this a bug or something I can solve myself? If the latter, how may I do that?

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -33,7 +33,7 @@ futures-core = "0.3.21"
 http = "0.2.6"
 http-body = "0.4.4"
 parking_lot = "0.12.0"
-sqlx = { version = "0.5.11", default-features = false }
+sqlx = { version = "0.6", default-features = false }
 thiserror = "1.0.30"
 tower-layer = "0.3.1"
 tower-service = "0.3.1"

